index.html

```diff
@@ -6,18 +6,80 @@
   <title>The HTML5 Herald</title>
   <meta name="description" content="The HTML5 Herald">
   <meta name="author" content="SitePoint">
-
+  <meta name="viewport" content="width=device-width">
   <style>
+      body {
+        margin: 0;
+      }
+      button {
+        width: 33%;
+        height: 80px;
+        background-color: gray;
+        border: 1px solid darkblue;
+        border-radius: 5px;
+        outline: none;
+      }
+      button:active {
+        background-color: #272727;
+      }
       #game {
-          width: 600px;
-          height: 600px;
+        width: 100vw;
+        height: 100vw;
```

Comment:
I think [`vmin`](https://developer.mozilla.org/en-US/docs/Web/CSS/length) would work here to keep the screen in frame regardless of aspect ratio, and might remove the need for the `@media` usages below

Reply:
ah nice. changed